### PR TITLE
PCLConfig.cmake.in edit for boost version to be same as the one in pcl_find_boost.cmake

### PR DESCRIPTION
**Pull request by al-tu**

### CODE REVIEW

**Review by mvieth — APPROVED** [review]
Thanks!

File: PCLConfig.cmake.in [hunk 1]
```diff
@@ -108,7 +108,7 @@ macro(find_boost)
     "1.59.0" "1.59" "1.58.0" "1.58" "1.57.0" "1.57" "1.56.0" "1.56" "1.55.0" "1.55")
```
Comment (kunaltyagi) [inline PCLConfig.cmake.in:108]: Let's remove this as well?

Comment (al-tu) [reply PCLConfig.cmake.in:108]: and in pcl_find_boost.cmake

Comment (al-tu) [reply PCLConfig.cmake.in]: done

**Review by kunaltyagi — APPROVED** [review]

**Review by mvieth — APPROVED** [review]

**Pull request merged**